config/routes.rb

```diff
@@ -400,6 +401,7 @@
 
     :cloud_object_store_container => {
       :get => %w(
+        dialog_edit
```

Comment:
@ZitaNemeckova why is this one dialog_edit but all others dialog_load, is that a typo?